### PR TITLE
Fix https://github.com/AdguardTeam/AdguardFilters/issues/84173

### PR DESCRIPTION
**Pull request by Yuki2718**
Way too generic. BTW is exclusion for ABP Japanese Filter still needed?

### CODE REVIEW

**Comment by Alex-302** [comment]
> is exclusion for ABP Japanese Filter still needed?

No.

File: filters/exclusions.txt [hunk 1]
```diff
@@ -257,5 +257,10 @@ _campaign=$popup
 ||github.io^$third-party,xmlhttprequest,domain=~facebook.com|~gametactic.org|~github.com|~github.io|~tumblr.com|~wottactic.com
 !###########################
 !
+!### NoCoin Filter List ###
+! https://github.com/AdguardTeam/AdguardFilters/issues/84173
+##.ad
```
Comment (Alex-302) [inline filters/exclusions.txt:262]: better regexp, like `/##\.an-advert-banner$/`

Comment (Alex-302) [reply filters/exclusions.txt:262]: without `^`

Comment (Yuki2718) [reply filters/exclusions.txt:262]: How about `/^##\.ad$/`?

**Comment by Yuki2718** [comment]
> 
> No.

Can I delete the part in this PR? Maybe it's better for you to do.

File: filters/exclusions.txt (hunk 1 above)
Comment (Yuki2718) [reply filters/exclusions.txt:262]: It seems `^` is not needed.

**Comment by Alex-302** [comment]
> Can I delete the part in this PR? Maybe it's better for you to do.

Yes, you can delete it. But check if they are not exists in another filters. In general this is not a big issue.

File: filters/exclusions.txt [hunk 2]
```diff
@@ -257,5 +257,10 @@ _campaign=$popup
 ||github.io^$third-party,xmlhttprequest,domain=~facebook.com|~gametactic.org|~github.com|~github.io|~tumblr.com|~wottactic.com
 !###########################
 !
+!### NoCoin Filter List ###
+! https://github.com/AdguardTeam/AdguardFilters/issues/84173
+/^##\.ad$/
```
Comment (Alex-302) [inline filters/exclusions.txt:262]: With `^` the rules like  `~example.com##.ad` will not be excluded.
Maybe better `/~.*##\.ad$/`

Comment (Alex-302) [reply filters/exclusions.txt:262]: I mean this rule: https://github.com/AdguardTeam/FiltersRegistry/blob/28ddd0b6c22c9d50eb7d6d1593eff80ea0461f62/filters/ThirdParty/filter_216_PolskiFiltr/filter.txt#L232

Comment (Yuki2718) [reply filters/exclusions.txt:262]: Okay.

Comment (Yuki2718) [reply filters/exclusions.txt:262]: I think `~` in the head should not be added.

**Comment by Yuki2718** [comment]
There are special mention about `_campaign=$popup` exclusion of ABPJP in Hebrew, French, and RU AdList.

File: filters/exclusions.txt [hunk 3]
```diff
@@ -257,5 +257,10 @@ _campaign=$popup
 ||github.io^$third-party,xmlhttprequest,domain=~facebook.com|~gametactic.org|~github.com|~github.io|~tumblr.com|~wottactic.com
 !###########################
 !
+!### NoCoin Filter List ###
+! https://github.com/AdguardTeam/AdguardFilters/issues/84173
+/.*##\.ad$/
```
Comment (Alex-302) [inline filters/exclusions.txt:262]: Checked again. The rule in Polish filter is limited to PL only - `***,pl##.ad`
But hufilter really has wide rule `##.ad`. So better revert to `/^##\.ad$/`

File: filters/exclusions.txt (hunk 2 above)
Comment (Alex-302) [reply filters/exclusions.txt]: `/.*##\.ad$/` will exclude domain specific rules.

Comment (Yuki2718) [reply filters/exclusions.txt:262]: > `/.*##\.ad$/` will exclude domain specific rules.

Oh, sure.

File: filters/exclusions.txt (hunk 3 above)
Comment (Yuki2718) [reply filters/exclusions.txt]: Done.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**